_episodes/04-pipefilter.md

```diff
@@ -522,23 +522,60 @@ so this matches all the valid data files she has.
 > {: .output}
 >
 > Explain why `-n` has this effect.
+>
+> > ## Solution
+> > The `-n` flag specifies a numeric sort, rather than alphabetical.
+> {: .solution}
 {: .challenge}
 
 > ## What Does `<` Mean?
 >
+> Change directory to data-shell (the top level of our downloaded example data)
+> ```
+> cd ~/data-shell
```

Comment:
Remove this `cd` command, as people may have `data-shell` in e.g., their Desktop